examples/adsense.amp.html

```diff
@@ -58,7 +58,7 @@
       id="amp-user-notification">
     This site uses cookies to personalize content.
     <a href="#learn-more">Learn more.</a>
-    <button on="tap:amp-user-notification.dismiss" role="button" tabindex="0">I accept</button>
+    <button on="tap:amp-user-notification.dismiss" tabindex="0">I accept</button>
```

Comment:
Buttons are also focusable, can we remove any `tabindex="0"`? (other #s are meaningful and can be left as is for providing focus order though)

Reply:
I can do this in a followup PR. This one touches a lot of files already.